cli/command/image/build.go

```diff
@@ -155,10 +155,8 @@ func NewBuildCommand(dockerCli command.Cli) *cobra.Command {
 	flags.SetAnnotation("stream", "version", []string{"1.31"})
 
 	flags.StringVar(&options.progress, "progress", "auto", "Set type of progress output (only if BuildKit enabled) (auto, plain, tty). Use plain to show container output")
-	flags.SetAnnotation("progress", "experimental", nil)
 
 	flags.StringArrayVar(&options.secrets, "secret", []string{}, "Secret file to expose to the build (only if BuildKit enabled): id=mysecret,src=/local/secret")
-	flags.SetAnnotation("secret", "experimental", nil)
```

Comment:
Wonder if we should have a new annotation to hide / disable these if the CLI (or daemon) is configured to use builder v1

Reply:
you mean hide the `progress` and `secret` flags?

Reply:
probably better when builder v1 is configured on the cli side

Reply:
Yes; not a blocker for now, but would be good to look into